lib/tasks/data.rake

```diff
@@ -0,0 +1,227 @@
+namespace :data do
+  desc "Creates departments and agencies"
+  task create_depts: :environment do
+    Department.all.each{ |d| d.home_node && d.home_node.delete }
+    Agency.all.each{ |a| a.home_node && a.home_node.delete }
+
+    Agency.delete_all
+    Department.delete_all
```

Comment:
Yeah - we have prod data we can't just blow away. 
One thing we need to do is to convert the existing Agency sections for depts into actual Departments.